modules/microshift-manifests-overview.adoc

```diff
@@ -4,12 +4,23 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="microshift-manifests-overview_{context}"]
-= How manifests work with kustomize
+= How kustomize works with manifests to deploy applications
 
-The `kustomize` configuration management tool is integrated with {microshift-short}. At every start, {microshift-short} searches the `/etc/microshift/manifests`, `/etc/microshift/manifests.d/++*++`, `/usr/lib/microshift/`, and `/usr/lib/microshift/manifests.d/++*++` manifest directories for a `kustomization.yaml`, `kustomization.yml`, or `Kustomization` file. If it finds one, {microshift-short} automatically runs the equivalent of the `kubectl apply -k` command to apply the identified manifests to the cluster.
+The `kustomize` configuration management tool is integrated with {microshift-short}. You can use Kustomize and the {oc-first} together to apply customizations to your application manifests and deploy those applications to a {microshift-short} cluster.
+
+* A `kustomization.yaml` file is a specification of resources plus customizations.
+* Kustomize uses a `kustomization.yaml` file to load a resource, such as application, then applies any changes you want to that application manifest and produces a copy of the manifest with the changes overlaid.
+* This operation keeps the original configuration file for your application intact, while enabling you to deploy iterations and customizations of your applications efficiently.
+* Using an `oc` command, you can then deploy the application in your {microshift-short} cluster.
+
+[id="how-microshift-uses-manifests"]
+== How {microshift-short} uses manifests
+At every start, {microshift-short} searches the `/etc/microshift/manifests`, `/etc/microshift/manifests.d/++*++`, `/usr/lib/microshift/`, and `/usr/lib/microshift/manifests.d/++*++` manifest directories for a `kustomization.yaml`, `kustomization.yml`, or `Kustomization` file. If it finds one, {microshift-short} automatically runs the equivalent of the `kubectl apply -k` command to apply the identified manifests to the cluster.
```

Comment:
Out of scope of this PR I think, but wondering if there is a way to make these lists easier to parse than in a running paragraph. Nothing obvious immediately occurs to me though :sweat_smile: